waiter/integration/waiter/basic_test.clj

```diff
@@ -344,62 +345,56 @@
           (is (pos? (get-in service ["service-description" "cpus"])) service)))
       (delete-service waiter-url service-id))))
 
-; Marked explicit due to:
-;   FAIL in (test-delete-app) (basic_test.clj:251)
-; test-delete-app service-deleted-from-all-routers
-; [CID=unknown] Expected status: 200, actual: 400
-; Body:{"exception":["Read timed out",
-; "java.net.SocketInputStream.socketRead0(Native Method)",
-; "java.net.SocketInputStream.socketRead(SocketInputStream.java:116)"
-; ...
-; "marathonclj.rest.apps$delete_app.invoke(apps.clj:17)",
-; "waiter.scheduler.marathon.MarathonScheduler$fn__18236.invoke(marathon.clj:260)",
-; "waiter.scheduler$retry_on_transient_server_exceptions_fn$fn__17306.invoke(scheduler.clj:91)"
-; ...
-; "waiter.scheduler.marathon.MarathonScheduler.delete_app(marathon.clj:257)",
-; "waiter.core$delete_app_handler.invokeStatic(core.clj:410)"
-; ...
-(deftest ^:parallel ^:integration-fast ^:explicit test-delete-app
+(deftest ^:parallel ^:integration-fast ^:dev test-delete-service
```

Comment:
Why `^dev`?

Reply:
Removed the `:dev`, I was using the tag to run only these tests on my parent branch that fixes all the explicit tests.